### PR TITLE
data: MyBio @PSGitHubUser1

### PR DESCRIPTION
**Pull request by PSGitHubUser1**
## Check List (Check all the applicable boxes) 

- [  ] My code follows the code style of this project.
- [  ] My change requires changes to the documentation.
- [  ] I have updated the documentation accordingly.
- [  ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3652"></a>

### CODE REVIEW

File: data/infoy_bio.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Infoy",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "world(!successfull=try)",
+  "avatar": "https://avatars.githubusercontent.com/u/90406016?s=400&u=65d5649db9b66f4c2478e322842d5d01aad3172a&v=4",
```
Comment (senali-d) [inline data/infoy_bio.json:6]: Replace avatar image as follows
```suggestion
  "avatar": "https://github.com/PSGitHubUser1.png",
```

File: data/infoy_bio.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/PSGitHubUser1",
+      "icon": "FaTwitter"
```
Comment (senali-d) [inline data/infoy_bio.json:11]: Use correct icon.
Further reference:  https://linkfree.eddiehub.io/icons
```suggestion
      "icon": "FaGitHub"
```

File: data/infoy_bio.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    {
+      "name": "Follow me on Youtube",
+      "url": "https://github.com/@infoyps",
+      "icon": "FaGitHub"
```
Comment (senali-d) [inline data/infoy_bio.json:16]: Please make sure url and icon correct.

**Review by senali-d — CHANGES_REQUESTED** [review]
Rename json file to github username.
`data/infoy_bio.json` -> `data/PSGitHubUser1.json`

**Review by senali-d — COMMENTED** [review]
Look Good!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**